packages/vscode-extension/src/handlers.ts

```diff
@@ -196,6 +208,21 @@ export async function activate(): Promise<Result<Void, FxError>> {
   return result;
 }
 
+async function refreshEnvTree(workspacePath: string, files: readonly Uri[]) {
```

Comment:
nit: how about change `refreshEnvTree` to `refreshEnvTreeOnFileChanged`?

Reply:
sure, updated.